lib/validators/modelValidator.ts

```diff
@@ -558,20 +558,37 @@ export class ModelValidator extends SpecValidator<SpecValidationResult> {
 
       const exampleResponseHeaders = exampleResponseValue[exampleResponseStatusCode].headers || {}
       const exampleResponseBody = exampleResponseValue[exampleResponseStatusCode].body
-      if (exampleResponseBody && !response.schema) {
+
+      // Fail when example provides the response body but the swagger spec doesn't define the schema for the response.
+      if (exampleResponseBody !== undefined && !response.schema) {
         const msg =
           `Response statusCode "${exampleResponseStatusCode}" for operation ` +
           `"${operation.operationId}" has response body provided in the example, ` +
           `however the response does not have a "schema" defined in the swagger spec.`
         const e = this.constructErrorObject<Sway.ValidationEntry>({
-          code: C.ErrorCodes.ResponseSchemaNotInSpec,
+          code: C.ErrorCodes.ExtraResponseBodyInExample,
```

Comment:
`C.ErrorCodes.ResponseSchemaNotInSpec` is there to indicate the same thing `that a response is provided in example for which the schema is not defined in the swagger spec`. So please replace the code `ExtraResponseBodyInExample` with `ResponseSchemaNotInSpec`.